Xamarin.Forms.Platform.Android/Renderers/EntryRenderer.cs

```diff
@@ -279,5 +285,35 @@ void UpdateReturnType()
 			Control.ImeOptions = Element.ReturnType.ToAndroidImeAction();
 			_currentInputImeFlag = Control.ImeOptions;
 		}
+
+		void SelectionChanged(object sender, SelectionChangedEventArgs e)
+		{
+			var control = Control;
```

Comment:
Not sure why we're making a variable for `Control`.

Reply:
My understanding was that the getter for Control makes non-trivial calls to get the native component (however I cannot see that looking at the ViewRenderer code :-)). So if we need to access it more than once it's a good idea for performance to save it in a variable for later use. If this is not how we wish to do it or if I misunderstood just let me know and I'll change it back.

Reply:
i would add just a null check to the `control` , and use `ElementController?.SetValueFromRenderer`